lib/rust/parser/src/syntax/tree.rs

```diff
@@ -194,11 +230,140 @@ impl<'s> span::Builder<'s> for Error {
 }
 
 
+// == Code Blocks ===
+
+/// A line of code.
+#[derive(Debug, Clone, PartialEq, Eq, Visitor, Reflect, Serialize, Deserialize)]
+pub struct Line<'s> {
```

Comment:
This should either be named `CodeBlockLine` or should be moved to `code_block` module. Right now, we have `syntax::tree::Line` and it doesnt make sense, as this is not "AST Line".